expo-zustand-styled-components/src/components/FileTree/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './FileTree';
```

Comment:
I am guessing this page was not formatted with prettier?
It needs an extra line